jwst/ramp_fitting/ramp_fit.py

```diff
@@ -336,11 +336,9 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
             opt_res.cr_mag_seg[num_int, :, rlo:rhi, :] = \
                        data_diff * (dq_cr != 0)
 
-    # Compute the final 2D array of differences.
+    # Compute the final 2D array of differences; create rate array
     median_diffs_2d /= n_int
-    med_slopes = median_diffs_2d.copy()
-
-    del median_diffs_2d
```

Comment:
Do you still need the median_diffs_2d array later, or could you delete it, as had been done before?